command/create.go

```diff
@@ -33,8 +33,8 @@ var CreateIssueCommand = flyte.Command{
 func createIssueHandler(input json.RawMessage) flyte.Event {
 	var handlerInput struct {
 		Project   string `json:"project"`
-		IssueType string `json:"issueType"`
-		Title     string `json:"title"`
+		IssueType string `json:"issuetype"`
```

Comment:
This is changing existing command (basically all updates to the json fields below). All existing flows using it will be affected by this change.

Are there any flows using it? Is it something you've planned to address?

Reply:
done